common/core/src/main/java/com/vmware/connectors/common/payloads/response/CardActionInputField.java

```diff
@@ -240,4 +243,26 @@ private boolean validate(CardActionInputField field) {
             return true;
         }
     }
+
+    public String hash() {
+        final List<String> optionList = new ArrayList<>();
+        if (!CollectionUtils.isEmpty(this.options)) {
+            final Map<String, String> sortedMap = new TreeMap<>(this.options);
+            optionList.add(sortedMap.toString());
+        }
+
+        return HashUtil.hash(
+                "id: ", this.id,
+                "label: ", this.label,
+                "format: ", this.format,
+                "options: ", optionList.toString(),
+                "minLength: ", String.valueOf(this.minLength),
```

Comment:
You shouldn't need to do this if you change the HashUtil to take in `Object... args`.